QuantConnect.DataBento/DataBentoDataProvider.cs

```diff
@@ -34,46 +35,20 @@ namespace QuantConnect.Lean.DataSource.DataBento
     /// </summary>
     public class DataBentoProvider : IDataQueueHandler
     {
-        /// <summary>
-        /// <inheritdoc cref="IDataAggregator"/>
-        /// </summary>
         private readonly IDataAggregator _dataAggregator = Composer.Instance.GetExportedValueByTypeName<IDataAggregator>(
             Config.Get("data-aggregator", "QuantConnect.Lean.Engine.DataFeeds.AggregationManager"), forceTypeNameOnExisting: false);
-
-        /// <summary>
-        /// <inheritdoc cref="EventBasedDataQueueHandlerSubscriptionManager"/>
-        /// </summary>
         private EventBasedDataQueueHandlerSubscriptionManager _subscriptionManager = null!;
-
         private readonly List<SubscriptionDataConfig> _activeSubscriptionConfigs = new();
-
-        private readonly System.Collections.Concurrent.ConcurrentDictionary<Symbol, SubscriptionDataConfig> _subscriptionConfigs = new();
-
-        /// <summary>
-        /// <inheritdoc cref="DatabentoRawClient"/>
-        /// </summary>
+        private readonly ConcurrentDictionary<Symbol, SubscriptionDataConfig> _subscriptionConfigs = new();
         private DatabentoRawClient _client = null!;
-
-        /// <summary>
-        /// DataBento API key
-        /// </summary>
         private readonly string _apiKey;
-
-        /// <summary>
-        /// DataBento historical data downloader
-        /// </summary>
         private readonly DataBentoDataDownloader _dataDownloader;
-
-        private bool _unsupportedSecurityTypeMessageLogged;
-        private bool _unsupportedDataTypeMessageLogged;
         private bool _potentialUnsupportedResolutionMessageLogged;
-
         private bool _sessionStarted = false;
```

Comment:
This flag isn't working because you start the session after subscribing. 😥
> [docs](https://databento.com/docs/api-reference-live/client/start?historical=python&live=python&reference=python)

<img width="645" height="251" alt="Image" src="https://github.com/user-attachments/assets/ae711e07-eb95-408b-ab0b-e198467a44eb" />